globus_automate_client/action_client.py

```diff
@@ -131,6 +131,16 @@ def status(self, action_id: str) -> GlobusHTTPResponse:
 
         return self.get(f"{quote(action_id)}/status")
 
+    def get_definition(self, action_id: str) -> GlobusHTTPResponse:
+        """
+        Query the Action Provider for the status of executed Action
+
+        :param action_id: An identifier that uniquely identifies an Action
+            executed on this Action Provider.
+        """
```

Comment:
I recognize that the class hierarchy here is all kinds of misleading, and that this is both a client for APs and-also the client for Specific Flows.

But what should we put in the docstring here? (Right now looks like a copy-paste)

I'm half-tempted to say that the docstring just says "for Flows only" and nothing more?

Reply:
You're correct, this was copy-and-paste. It took all of two minutes to create IIRC.

I can add some text that says "for Flows only". I'm leery of spending more than two additional minutes on this PR, though, so heads-up that I hold that opinion!